lib/vsc/install/shared_setup.py

```diff
@@ -268,7 +268,20 @@ def build_setup_cfg_for_bdist_rpm(target):
 def action_target(target, setupfn=setup, extra_sdist=[]):
     # EXTRA_SDIST_FILES.extend(extra_sdist)
 
-    cleanup()
+    do_cleanup = True
+    try:
+        # very primitive check for install --skip-build
+        # in that case, we don't mind "leftover build";
+        # it's probably intentional
+        install_ind=sys.argv.index('install')
+        build_skip=sys.argv.index('--skip-build')
```

Comment:
style: ` = ` (spaces), otherwise good to go I guess (I can't think of a better way unless changing the `setup.py` significantly more)